src/main/java/com/gachigang/ontherun/controller/CategoryController.java

```diff
@@ -0,0 +1,46 @@
+package com.gachigang.ontherun.controller;
+
+import com.gachigang.ontherun.common.mapper.CategoryMapper;
+import com.gachigang.ontherun.payload.category.request.CreateCategoryRequest;
+import com.gachigang.ontherun.payload.category.request.UpdateCategoryRequest;
+import com.gachigang.ontherun.payload.category.response.CategoryResponse;
+import com.gachigang.ontherun.service.CategoryService;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequiredArgsConstructor
+@Slf4j
```

Comment:
I don`t thing that we need it here, since we don`t log anything

Reply:
Already fixed it